modules/siddhi-core/src/main/java/io/siddhi/core/query/processor/stream/AbstractStreamProcessor.java

```diff
@@ -78,6 +79,7 @@ public void initProcessor(MetaStreamEvent metaStreamEvent,
             StateFactory<S> stateFactory = init(metaStreamEvent, metaStreamEvent.getLastInputDefinition(),
                     attributeExpressionExecutors, configReader, streamEventClonerHolder, outputExpectsExpiredEvents,
                     findToBeExecuted, siddhiQueryContext);
+//            ProcessorAnnotationValidator.validateAnnotation(this, attributeExpressionExecutors);
```

Comment:
Please remove these unnecessary lines and clean the code

Reply:
This is where the validation class is called